.gitignore

```diff
@@ -501,3 +501,9 @@ workshops/eventmesh-teq-kafka/**/deploy/
 workshops/eventmesh-teq-kafka/cloud-setup/database/
```

Comment:
I think these are, and if not should be, captured at the root level gitignore?

Reply:
yes 👍🏻 
from root .gitignore
## Remove Other Local project stuffs
workshops/eventmesh-teq-kafka/cloud-setup/database/